evi-next-js-function-calling/.env.example

```diff
@@ -0,0 +1,4 @@
+HUME_API_KEY=""
+HUME_CLIENT_SECRET=""
```

Comment:
Unless I am mistaken, I think we should also prefix these env variables with `NEXT_PUBLIC_`

https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables#bundling-environment-variables-for-the-browser